main.go

```diff
@@ -41,8 +40,8 @@ func main() {
 		"サムタブロ修道院",
 		"サムター要塞",
 		"サムルノリ",
+		"サム・アルトマン",
 	}
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Since Go 1.20, rand.Seed() has been deprecated.
https://tip.golang.org/doc/go1.20#mathrandpkgmathrand
> The [math/rand](https://tip.golang.org/pkg/math/rand/) package now automatically seeds the global random number generator (used by top-level functions like Float64 and Int) with a random value, and the top-level [Seed](https://tip.golang.org/pkg/math/rand/#Seed) function has been deprecated. Programs that need a reproducible sequence of random numbers should prefer to allocate their own random source, using rand.New(rand.NewSource(seed)).

Reply:
good catch, thanks!